.github/workflows/smoke-test.yml

```
@@ -193,6 +193,11 @@ jobs:
         # (Even in unittests we should use "importskip" rather than "skipif".)
         run: coverage run -m pytest -v ${{ ! contains(matrix.extra-install, 'polars') && '-k "not lazyframe and not polars and not pca"' || ''}}
 
+      - name: Test replace-binary-path
+        if: ${{ contains(matrix.extra-install, 'polars') }}
+        # Messes with envvars, so can't be run in parallel!
+        run: FORCE_TEST_REPLACE_BINARY_PATH=1 pytest -k 'test_replace_binary_path'
```

```suggestion
        run: coverage run -m pytest -v ${{ ! contains(matrix.extra-install, 'polars') && '-k "not lazyframe and not polars and not pca"' || 'not test_replace_binary_path'}}

      - name: Test replace-binary-path
        if: ${{ contains(matrix.extra-install, 'polars') }}
        # Messes with envvars, so can't be run in parallel!
        run: pytest -k 'test_replace_binary_path'
```
Maybe just use `-k` instead of introducing a new envvar?

---

I want to avoid needing "special -k" configuration on the happy path. Without the env var, `pytest -n auto` fails.